installers/mac/tar/build.gradle

```diff
@@ -57,6 +57,7 @@ task configureBuild(type: Exec) {
     // Platform specific flags
     def command = ['bash', 'configure',
             "--with-toolchain-type=clang",
+            "--with-zlib=system",
```

Comment:
Add this to the changelog.

Reply:
Ben, just wondering if this line is essential? I checked `generated-configure.sh` at L51372. The default zlib pick is system:
```
  if test "x$OPENJDK_TARGET_OS" = xmacosx; then
    #
    # On macosx default is system...on others default is
    #
    DEFAULT_ZLIB=system
  fi
```

The reason I prefer not to have it is to keep the macOS configuration flags consistent between 8, 11 and later versions.

Reply:
I think it was leftover from previous testing. Removed.